frontend/packages/ceph-storage-plugin/integration-tests/tests/storage-dashboard.scenario.ts

```diff
@@ -0,0 +1,34 @@
+import { browser } from 'protractor';
+import { appHost } from '@console/internal-integration-tests/protractor.conf';
+import { dashboardIsLoaded } from '../../../console-shared/src/test-views/dashboard-shared.view';
+import { serviceName, clusterHealth, clusterName, allNodes } from '../views/storage-dashboard.view';
+import { execSync } from 'child_process';
+
+describe('Check data on Storage Dashboard.', () => {
+  beforeAll(async () => {
+    await browser.get(`${appHost}/dashboards/persistent-storage`);
+    await dashboardIsLoaded();
+  });
+
+  it('Check cluster health is OK', async () => {
+    expect(clusterHealth.getText()).toContain('is healthy');
```

Comment:
`healthy` should be good enough. 
Also can you lowercase of the text so it doesn't fail even if we change the case of any letter.
Can we define constants somewhere on top of file?
```suggestion
    expect(clusterHealth.getText()).toContain('healthy');
```

Reply:
+1 on making constants for the strings getting used.

Reply:
Changed to 'healthy' and made it a constant. Couldn't find an elegant solution of converting clusterHealth.getText() promise to lowercase, as clusterHealth.getText().toLowerCase() doesn't work. I'm not sure it's necessary, either, because 4.2 is past feature freeze and in 4.3 there's no word 'healthy' at all, just the words 'OCS Cluster' instead of the cluster name and the green tick

Reply:
Since this is targetting master, I would expect the code to work with master (4.3) rather than already released 4.2

Reply:
BTW, just to clarify, in order to use toLowerCase(), you can use parentheses 
`(await clusterHealth.getText()).toLowerCase()`